api/app/helpers/sendAlert.ts

```diff
@@ -1,27 +1,54 @@
 import axios from "axios";
+import { Logger } from "./logger";
 require("dotenv").config();
 
+function transformUrl(transcriptUrl?: string | null): string | null {
+  if (!transcriptUrl) {
+   Logger.error("Error transforming URL: URL not found");
+    return null;
+  }
+
+  // Extract necessary parts from the transcriptUrl
+  const { pathname } = new URL(transcriptUrl);
+  const transcriptPath = pathname.split("/").slice(5).join("/");
+
+  // Construct the correct URL using template literals
+  const transcriptUrlPath = `https://btctranscripts.com/${transcriptPath}`;
+
+  // Remove .md from the end of the URL
+  const transformedUrlPath = transcriptUrlPath.replace(/\.md$/, "");
+
+  return transformedUrlPath;
+}
+
 export async function sendAlert(
   message: string,
+  isError: boolean = false,
   transcriptTitle?: string | null,
+  speakers?: string | null,
   transcriptUrl?: string | null,
-  transcriptHash?: string | null
 ) {
-
- //bypass alerts in development
+  //bypass alerts in development
   if (process.env.NODE_ENV === "development") {
     return;
   }
 
-  if (!process.env.DISCORD_WEBHOOK_URL ) {
-    throw new Error("DISCORD_WEBHOOK_URL is not set");
+  // Use ternary operator for setting webhookUrl
+  const webhookUrl = isError ? process.env.DISCORD_ERROR_WEBHOOK_URL : process.env.DISCORD_TRANSCRIPT_WEBHOOK_URL;
+
+  if (!webhookUrl) {
+    Logger.error("Error sending alert: Webhook URL not found");
+    return;
   }
 
-  const content = `New Queuer Action\nMessage: ${message}\nTranscript Title: ${transcriptTitle}\nTranscriptUrl: ${transcriptUrl}\ntranscriptHash: ${transcriptHash}`;
+  const transformedUrl = transformUrl(transcriptUrl);
```

Comment:
When `sendAlert` is called with an error (`isError = true`) there is no `transcriptUrl`, so `transformUrl` always fails.

You can just skip this calculation when there is an error.